web-services/boards.test.js

```diff
@@ -0,0 +1,11 @@
+var request = require('./request');
+//url is contained /boards endpoint that deployed on AWS lambda 
+var url = 'https://mpau0ppws1.execute-api.us-east-1.amazonaws.com/dev/boards';
```

Comment:
This isn't going to work when the API is deployed again since the URLs aren't guaranteed.

Reply:
I appreciate your comment. I misunderstood the test. I thought the test request how to make a API request directly to test a network endpoint like AWS lambda in jest. After I re-read "the research how to mock API  request in jest", I understood how the code that made a fake request in the example work. But still unclear how to apply an idea to the test for the board endpoint. Hopefully, after discuss on Monday, I will have a right way to do it. Thank you